packages/jest-expo/src/preset/setup-react-19.js

```diff
@@ -0,0 +1,5 @@
+// This is a temporary workaround for `react-test-renderer@19+`.
+// The renderer is deprecated, but there is no replacement for React Native yet.
+// Setting this global property disables the deprecation warning, and changes RTR to legacy mode.
+// See: https://discord.com/channels/514829729862516747/514832743654228009/1266527040480477274
+globalThis.IS_REACT_NATIVE_TEST_ENVIRONMENT = true;
```

Comment:
We could also define this in the `expo-module-scripts` preset, if we don't want to ship this temporary workaround to users.

Reply:
Edit: I'll pull it out of `jest-expo` and into `expo-module-scripts`, we need a user-facing solution eventually and this is just a temporary workaround.